SWLOR.Game.Server/Event/Conversation/Tatooine GoCorp Transport/tp_gocorp_mos.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NWN;
+using SWLOR.Game.Server.Data.Contracts;
+using SWLOR.Game.Server.GameObject;
+using SWLOR.Game.Server.Service.Contracts;
+using Object = NWN.Object;
+
+namespace SWLOR.Game.Server.Event.Conversation
+{
+    public class tp_gocorp_mos : IRegisteredEvent
```

Comment:
Can you use a more descriptive name for this class? Only the entry points are restricted by the 16 characters. If this is in the SWLOR.Game.Server.Event.Conversation namespace it shouldn't be an entry point. The entry points are only in SWLOR.Game.Server.NWN.Events.Conversation - I know this is confusing.